src/android/InAppBrowser.java

```diff
@@ -129,6 +131,8 @@ public class InAppBrowser extends CordovaPlugin {
 
     private static final String BANNER_TAPPED_EVENT = "bannertapped";
 
+    private static final String SHOW_NAVIGATION_BAR = "navigationbar";
```

Comment:
given this does more than just show the navigation bar, should we change the variable name here?

Reply:
I changed `navigationbar` to `pagetitleheader`